schemas/vault-config/replication-1.yml

```diff
@@ -0,0 +1,24 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /vault-config/replication-1.yml
+
+  provider:
+    type: string
```

Comment:
this should be implemented via a `oneOf` so that we have a real provider pattern.